src/System.IO.FileSystem/src/System/IO/Win32FileStream.cs

```diff
@@ -2048,6 +1797,21 @@ public override Task FlushAsync(CancellationToken cancellationToken)
                 return Task.CompletedTask;
         }
 
-        private struct VoidTaskResult { }
+        private static int WaitAndUnwrapException(Task<int> task)
```

Comment:
I did not realize that this was a thing, but I am very happy that it does exist!
Done